tests/test_diffeq/test_odefiltsmooth/test_kalman_odesolution.py

```diff
@@ -137,19 +139,20 @@ def test_output_shape(self):
                     else:
                         self.assertEqual(sample.shape, size + loc_shape)
 
-    def test_sampling_two_locations_multiple_samples(self):
-        locs = self.solution.t[[2, 3]]
-        five_samples = self.solution.sample(t=locs, size=5)
-
-        chi_squared = np.array(
-            [
-                chi_squared_statistic(
-                    sample,
-                    self.solution[:].mean[[2, 3]],
-                    self.solution[:].cov[[2, 3]],
-                )
-                for sample in five_samples
-            ]
-        ).mean()
-        self.assertLess(chi_squared, 10.0)
-        self.assertLess(0.1, chi_squared)
+    #
+    # def test_sampling_two_locations_multiple_samples(self):
```

Comment:
This should probably be removed before merging

Reply:
This is related to the discussion about "what does this test do" in #306 

In any case, I removed it.